src/emulator.py

```diff
@@ -252,34 +253,79 @@ def read_and_confirm_mnemonic() -> None:
         client.input(mnemonic[index])
         time.sleep(SLEEP)
 
+    # Click Continue to finish the quiz
     client.press_yes()
+    time.sleep(SLEEP)
+
+    # Click Continue to finish the backup
     client.press_yes()
+    time.sleep(SLEEP)
+
     client.close()
 
 
-def read_and_confirm_mnemonic_shamir():
+def read_and_confirm_mnemonic_shamir(shares: int = 1, threshold: int = 1):
+    # TODO: decide what numbers will be allowed (there are some maximum numbers)
+    if shares < 1 or shares > 6:
```

Comment:
As [per the spec](https://github.com/satoshilabs/slips/blob/master/slip-0039.md) there is at most 16 shares possible.

Btw the spec also allows something called SuperShamir. This is not however supported by Suite and won't be anytime soon. Could you still please share a note that "This function does not support Super Shamir"?